be/src/olap/rowset/segment_v2/column_writer.h

```diff
@@ -140,8 +142,7 @@ class ColumnWriter {
     rowid_t _next_rowid = 0;
 
     const EncodingInfo* _encoding_info = nullptr;
-    // const CompressionCodec* _codec = nullptr;
-    // TODO(zc): compression type
+    const BlockCompressionCodec* _compress_codec = nullptr;
```

Comment:
This field is not needed. Just pass `_opts.compression_type` into PageCompressor and let it retrieve BlockCompressionCodec.

Reply:
I prefer to keep this field here. If we keep it here, we will find codec only once, otherwise we will retrieve it for each page.

Reply:
OK